tests/jest/webpack/loaderUtils/testContent/packages/module/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@cerner/testing"
+
```

Comment:
:eyes:

Reply:
just a fake package.json for jest testing.